lib/sentry/config.ex

```diff
@@ -197,6 +197,14 @@ defmodule Sentry.Config do
       A map of tags to be sent with every event.
       """
     ],
+    extra: [
+      type: {:map, :atom, :any},
```

Comment:
@Gladear this breaks a bunch of tests but we want extra to be tagged as atoms. Can you fix the tests?

Reply:
Done!